PThreads on Windows build

I'm building SRT library in VS 2013, but there is a recommendation to not do that with pThreads. I'm wondering what would be the reason / impact of building SRT lib with pThreads support. Is there an explanation for that?
-- Thanks!

Understood. However in VS 2013 (12.0.40629.00 Update 5) does not complete the compilation of SRT Lib with anything else, but PThreads. As if VS 2013 is not fully compliant with C++11 standard on Windows 10 [10.0.19043.1165]

Below the build script (on windows):
----------------------------------------
git clone https://github.com/microsoft/vcpkg.git
cd .\vcpkg
.\bootstrap-vcpkg.bat
set VCPKG_ROOT=%cd%
vcpkg install openssl --triplet x64-windows
vcpkg integrate install
vcpkg install pthreads --triplet x64-windows
vcpkg integrate install
cd ..
git clone -b v1.4.3 https://github.com/Haivision/srt.git ./v1.4.3
cd .\v1.4.3
cmake ./ -G "Visual Studio 12 2013" -A x64 -DENABLE_STDCXX_SYNC=ON -DCMAKE_TOOLCHAIN_FILE=%VCPKG_ROOT%/scripts/buildsystems/vcpkg.cmake -DCMAKE_BUILD_TYPE=Release
cmake --build . --config Release

Build log:
----------
sync.cpp(122): error C2511: 'bool srt::sync::CEvent::lock_wait_until(const std::chrono::time_point<srt::sync::steady_clock,std::chrono::duration<std::chrono::syste
m_clock::rep,std::chrono::system_clock::period>> &)' : overloaded member function not found in 'srt::sync::CEvent' 
          e:\teste\haivision2\v1.4.3\srtcore\sync.h(556) : see declaration of 'srt::sync::CEvent'
sync.cpp(123): error C2597: illegal reference to non-static member 'srt::sync::CEvent::m_lock' 
sync.cpp(124): error C2228: left of '.wait_until' must have class/struct/union
sync.cpp(184): error C2511: 'bool srt::sync::CTimer::sleep_until(std::chrono::time_point<srt::sync::steady_clock,std::chrono::duration<std::chrono::system_clock::r
ep,std::chrono::system_clock::period>>)' : overloaded member function not found in 'srt::sync::CTimer' 
          e:\teste\haivision2\v1.4.3\srtcore\sync.h(609) : see declaration of 'srt::sync::CTimer'
E:\teste\haivision2\v1.4.3\srtcore\sync.cpp(187): error C2228: left of '.mutex' must have class/struct/union 
sync.cpp(188): error C2597: illegal reference to non-static member 'srt::sync::CTimer::m_tsSchedTime' 
sync.cpp(189): error C2228: left of '.mutex' must have class/struct/union 
sync.cpp(203): error C2440: 'initializing' : cannot convert from 'std::chrono::system_clock::time_point' to 'std::chrono::time_point<srt::sync::steady_clock,std::c
hrono::duration<std::chrono::system_clock::rep,std::chrono::system_clock::period>>'
          No constructor could take the source type, or constructor overload resolution was ambiguous
sync.cpp(205): error C2597: illegal reference to non-static member 'srt::sync::CTimer::m_tsSchedTime'
sync.cpp(205): fatal error C1903: unable to recover from previous error(s); stopping compilation 
  sync_cxx11.cpp
sync_cxx11.cpp(97): error C2146: syntax error : missing ';' before identifier 'CUDTException' 
sync_cxx11.cpp(97): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int

--- a/docs/build/build-win.md
+++ b/docs/build/build-win.md
@@ -67,7 +67,8 @@ SRT as of v1.4.2 supports two threading libraries:
 
 The `pthreads` library is provided out-of-the-box on all POSIX-based systems.
 On Windows it can be provided as a 3rd party library (see below).
-However the C++ standard thread library is recommended to be used on Windows.
+However, using pthread wrapper on Windows comes with limitations, e.g. converting time from POSIX-based to Windows-based might loose precision, monotonic clock implementation eventually uses wall clock, etc.  
+**The C++11 standard thread library is recommended to be used on Windows.**
 
 ### 1.3. Package Managers
 

